### DISCUSSION TITLE
How to get started with StrictDoc: Web server recommendations?

### DISCUSSION THREAD

**Discussion by alankritaastha16** [strictdoc-project/strictdoc#885] [Q&A] [answered]
hello,

I am new to strictdoc and i have a few questions to get started with it.

i need to generate some sample requirements using strict doc and publish them in a webserver. 
my questions are which webserver is recommended? which language is recommended for the webserver?
which git (flavour) is recommended??
how do i start too write a code to generate req using strictdoc?

**Comment by stanislaw** [comment, marked as answer]
I have added my answer to you in Discord to the docs/FAQ: https://strictdoc.readthedocs.io/en/latest/strictdoc_02_faq.html#which-web-server-is-recommended-for-strictdoc-documentation. That section now has some basic web server recommendations.

When it comes to Git (flavour) recommendation, it is entirely up to a user how to organize their workflow. Let me know if you are missing any specific detail.
